tests/net/socket_zep/tests/01-run.py

```diff
@@ -51,9 +51,10 @@ def testfunc(child):
 
 
 if __name__ == "__main__":
-    os.environ['TERMFLAGS'] = "--eui64=00:5a:45:50:0a:00:30:38 -z [%s]:%d,[%s]:%d" % (
-            zep_params['local_addr'], zep_params['local_port'],
-            zep_params['remote_addr'], zep_params['remote_port'])
+    os.environ['TERMFLAGS'] = "--eui64=00:5a:45:50:0a:00:30:38 -z [%s]:%d,[%s]:%d" % (zep_params['local_addr'],
+                                                                                      zep_params['local_port'],
+                                                                                      zep_params['remote_addr'],
+                                                                                      zep_params['remote_port'])
```

Comment:
Can you flake8 this please? I have the feeling this will be too long (+ I the change seems unrelated).

Reply:
You're right - it's unrelated (or rather a result of a back-and-forth). Flake seems to be cool with it, but I will revert it anyway.